src/examples/hid-keyboard/main.go

```diff
@@ -19,3 +27,23 @@ func main() {
 		}
 	}
 }
+
+func init() {
+	if usbVID != "" {
+		vid, _ := strconv.Atoi(usbVID)
+		usb.VendorID = uint16(vid)
+	}
+
+	if usbPID != "" {
+		pid, _ := strconv.Atoi(usbPID)
+		usb.ProductID = uint16(pid)
+	}
```

Comment:
Since hexadecimal numbers are used more often than decimal numbers, strconv.ParseUINT(x, 16, 0) may be a better choice.

Reply:
good point. done.